web_src/less/_base.less

```diff
@@ -980,7 +980,7 @@ a.commit-statuses-trigger {
 
 .ui.cards > .card > .content,
 .ui.card > .content {
-  border-color: var(--color-secondary);
+  border: none;
```

Comment:
This will almost definitely have side-effects elsewhere, like on the user profile page.